cypress/integration/brewery/ScenarioParameters-additional-advanced-tests.spec.js

```diff
@@ -0,0 +1,169 @@
+    // Delete all tests scenarios
+    ScenarioManager.switchToScenarioManager();
+    for (const scenarioName of scenarioNamesToDelete) {
+      ScenarioManager.deleteScenario(scenarioName);
+    }
+  });
+
+  it('addtional advanced scenario parameters tests', () => {
+    const INIT_VALUES = {
+      additionalSeats: -4,
+      evaluation: 'Good',
+      volumeUnit: 'L',
+      addtionalTables: 3,
+      comment: 'None',
+      additionalDate: '06/22/2022',
+    };
+    const VALUES_TO_UPDATE = {
+      additionalSeats: 888,
+      evaluation: 'Awful',
+      volumeUnit: 'bl',
+      addtionalTables: 9090,
+      comment: 'Strongly recommended',
+      additionalDate: '06/07/2029',
+    };
+
+    const scenarioName = forgeScenarioName();
+    scenarioNamesToDelete.push(scenarioName);
+    Scenarios.createScenario(scenarioName, true, SCENARIO_DATASET, SCENARIO_RUN_TEMPLATE);
+
+    BreweryParameters.switchToEventsTab();
+    BreweryParameters.getAdditionalSeatsInput().should('value', INIT_VALUES.additionalSeats);
+    BreweryParameters.getActivatedInput().should('not.be.checked');
+
+    BreweryParameters.switchToAdditionalTab();
+    BreweryParameters.getVolumeUnitTextField().should('have.text', INIT_VALUES.volumeUnit);
+    BreweryParameters.getAdditionalTablesInput().should('value', INIT_VALUES.addtionalTables);
+
+    BreweryParameters.switchToEventsTab();
+    BreweryParameters.getEvaluationInput().should('value', INIT_VALUES.evaluation);
+
+    BreweryParameters.switchToAdditionalTab();
+    BreweryParameters.getCommentInput().should('value', INIT_VALUES.comment);
+    BreweryParameters.getAdditionalDateInput().should('value', INIT_VALUES.additionalDate);
+
+    ScenarioParameters.edit();
+
+    BreweryParameters.switchToEventsTab();
+    BreweryParameters.getAdditionalSeatsInput().click().clear().type(-800).should('value', -80);
+    BreweryParameters.getAdditionalSeatsInput().click().clear().type(200).should('value', 200);
+    BreweryParameters.getAdditionalSeatsInput().click().clear().type(-1000).should('value', -100);
+    BreweryParameters.getAdditionalSeatsInput().click().clear().type(3000).should('value', 300);
+    BreweryParameters.getAdditionalSeatsInput().click().clear().type(2000).should('value', 2000);
+
+    BreweryParameters.switchToAdditionalTab();
+    BreweryParameters.getVolumeUnitTextField().type('m {enter}').should('have.text', 'm³');
+
+    BreweryParameters.switchToEventsTab();
+    BreweryParameters.getActivatedInput().check().should('be.checked');
+
+    BreweryParameters.switchToAdditionalTab();
+    BreweryParameters.getAdditionalTablesInput().click().clear().type(-250).should('value', -25);
+    BreweryParameters.getAdditionalTablesInput().click().clear().type(-120).should('value', -120);
+    BreweryParameters.getAdditionalTablesInput().click().clear().type(50000).should('value', 5000);
+    BreweryParameters.getAdditionalTablesInput().click().clear().type(6000).should('value', 6000);
```

Comment:
Why do we have
 * -80 instead of -800
 * -100 instead of -1000
 * 300 instead of 3000
 * -25 instead of -250

If these parameters have min & max values, could we add a comment here to have these values somewhere ?

(ideally, we should import these min/max values from config files, and compute the input value after applying the min/max constraints, but I'm not sure if we can easily read config values from cypress tests)